.github/workflows/aws_copilot_deployment.yml

```diff
@@ -12,9 +12,9 @@ on:
             type: string
             required: true
         run_e2e_tests:
-            description: "Run end-to-end (browser) tests; only works for the `test` environment"
+            description: "Run end-to-end (browser) tests; only works for the `dev` and `test` environments"
             type: boolean
-            default: false
+            default: true
```

Comment:
Step closer to accidentally run it on prod. :eyes:

Reply:
Actually.. you should make set `run_e2e_tests` to `false` for the prod job. Just in case.